corehq/sql_accessors/sql_templates/get_all_reverse_indices.sql

```diff
@@ -0,0 +1,12 @@
+DROP FUNCTION IF EXISTS get_all_reverse_indices(text[]);
+
+CREATE FUNCTION get_all_reverse_indices(case_ids text[]) RETURNS SETOF form_processor_commcarecaseindexsql AS $$
+BEGIN
+    RETURN QUERY
+    SELECT * FROM form_processor_commcarecaseindexsql AS case_index_table
+    WHERE
+      -- case_index_table.domain = domain_name AND -- TODO: Uncomment if CaseIndex ever uses its domain column
+      case_index_table.referenced_id = ANY(case_ids)
```

Comment:
Do we have a sense for the max number of case_ids that will be passed into this function? `ANY` is essentially a long `OR` statement, and `OR`s tend to be slow.

I don't think we should do this now (since it also increases complexity), but just to keep in mind - a faster way to do this kind of thing is to setup a temporary table with an indexed column for case_id, insert the case_ids into it, and then use a join to the temporary table to filter.  But that's overkill if the number of case_ids will always be small (i.e., < 10)